src/main/java/pl/miloszkajetan/serwisAukcyjny/User/UserRegistrationService.java

```diff
@@ -0,0 +1,25 @@
+package pl.miloszkajetan.serwisAukcyjny.User;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.stereotype.Service;
+
+@Service
+public class UserRegistrationService {
+
+    @Autowired
+    private PasswordEncoder passwordEncoder; //pytanie 1
```

Comment:
passwordEncoder nie działa. Could not Autowire. No beans of "PasswordEncoder" were found.
Nie bardzo wiemy dlaczego.

Reply:
potrzeba wam beana który będzie robił za ten PasswordEncoder np
@Bean
PasswordEncoder getEncoder() {
    return new BCryptPasswordEncoder();
}

To się umieszcza w pliku z konfiguracja Springa.